sdk/compute/Azure.ResourceManager.Compute/samples/Generated/Samples/Sample_GalleryInVmAccessControlProfileVersionCollection.cs

```diff
@@ -118,7 +118,7 @@ public async Task Get_GetAGalleryInVMAccessControlProfileVersion()
         public async Task GetAll_ListGalleryInVMAccessControlProfileVersionsInAGalleryInVMAccessControlProfile()
         {
             // Generated from example definition: specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/examples/galleryResourceProfileExamples/GalleryInVMAccessControlProfileVersion_ListByGalleryInVMAccessControlProfile.json
-            // this example is just showing the usage of "GalleryInVMAccessControlProfileVersions_ListByGalleryInVmAccessControlProfile" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "GalleryInVMAccessControlProfileVersions_ListByGalleryInVMAccessControlProfile" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `VM`, not `Vm`: https://github.com/Azure/azure-rest-api-specs/blob/95aa73b8831deb2940ac7067b8f296cd189a5fe1/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json#L2400